### DISCUSSION TITLE
Why the forks of architect deploy and inventory? Is this in govcloud?

### DISCUSSION THREAD

**Discussion by david-saeger** [nasa-gcn/gcn.nasa.gov#2778] [Q&A] [answered]
Curious why the overrides to architect deploy and inventory forks? Is this for supply chain security or did you have to make changes to these dependencies for some reason. The context for this question is I am trying to deploy https://github.com/nasa-gcn/americana-stack to govcloud and architect is failing on recognizing some govcloud namespace changes. Was wondering if these forks were necessary to afford this option for gcn

**Comment by dakota002** [comment, marked as answer]
Hi @david-saeger, are you referring to this section in the package.json?

```json
"overrides": {
    "@architect/deploy": "github:DocLM/deploy#d3116f41d5fda00337ebfda543a69d3b3d4546be",
    "@architect/inventory": "github:lpsinger/inventory#esm-graph-with-top-level-await"
  },
```

These are pointing to commits that have updates or fixes we've found to be necessary for our deployment but have not yet been implemented into the original repositories.

There are currently PRs open for these though, https://github.com/architect/deploy/pull/389 and https://github.com/architect/inventory/pull/77 respectively

**Reply by david-saeger** [reply]
Ahh thank you for that. Very helpful. That mostly answers my question
